pkgs/applications/radio/qradiolink/default.nix

```diff
@@ -1,51 +1,64 @@
-{ stdenv, fetchFromGitHub, alsaLib, boost
-, qt4, libpulseaudio, codec2, libconfig
-, gnuradio, gr-osmosdr, gsm
-, libopus, libjpeg, protobuf, qwt, speex
-} :
+{ stdenv
+, mkDerivation
+, fetchFromGitHub
+, libpulseaudio
+, libconfig
+# Needs a gnuradio built with qt gui support
+, gnuradio
+, gnuradioPackages
```

Comment:
I think its best to avoid passing in both the main package and its package set, to avoid risking incompatible versions